<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookQueryWidget</class>
 <widget class="QWidget" name="BookQueryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>书名：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>出版年份：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="lowerYearSpinBox">
       <property name="maximum">
        <number>3000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>------</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="upperYearSpinBox">
       <property name="maximum">
        <number>3000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="searchPushButton">
     <property name="text">
      <string>搜索</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>出版社：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pressLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>状态：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="statusComboBox">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>默认</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>有库存</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>已被借出</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="clearPushButton">
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>作者：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="writerLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,2">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>类别：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="catagoryComboBox"/>
     </item>
    </layout>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="loanPushButton">
     <property name="text">
      <string>借阅</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QTableWidget" name="queryTableWidget">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>编号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>书名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>库存数量</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>入库时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>出版社</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>出版时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>类别</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>作者</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
